platooning_control/src/platooning_control.cpp

```diff
@@ -359,7 +359,7 @@ namespace platooning_control
     double dx1 = trajectory_points[trajectory_points.size()-1].x - trajectory_points[0].x;
     double dy1 = trajectory_points[trajectory_points.size()-1].y - trajectory_points[0].y;
     double d1 = sqrt(dx1*dx1 + dy1*dy1);
-    double t1 = rclcpp::Time((trajectory_points[trajectory_points.size()-1].target_time)).seconds() - rclcpp::Time(trajectory_points[0].target_time).nanoseconds();
+    double t1 = rclcpp::Time((trajectory_points[trajectory_points.size()-1].target_time)).seconds() - rclcpp::Time(trajectory_points[0].target_time).seconds();
```

Comment:
changing from nanosec to sec, has any impact somewhere else?
I dont remember if it does

Reply:
I kept this in seconds since were using seconds precision in the earlier implementation and the speed is being calculcated as m/s 
https://github.com/usdot-fhwa-stol/carma-platform/blob/11047f2bde3dc2f37b7d2744cd41a3d9772bbc42/platooning_control/src/platoon_control.cpp#L298

But using nanoseconds instead will not have any impact, probably better to do since rounding of mps can have greater impact on mph speed.